docs/www/guide-wasm-elastic.md

```diff
@@ -0,0 +1,189 @@
+`src/wasm.js`, and rename to `src/elastic.js`, remember
+the wasm file name is very important, from this we create
+the id for a wasm function. 
+
+we need the elastic search js client for this example:
+
+```bash
+npm install @elastic/elasticsearch
+```
+
+now we can edit `src/elastic.js` in order to apply our logic
+for ElasticSearch, you could use the next code.
+
+```js
+const {
+  SimpleTransform,
+  PolicyError,
+} = require("@vectorizedio/wasm-api");
+
+const {Client} = require('@elastic/elasticsearch')
+
+const client = new Client({ node: 'http://localhost:9200' })
+
+const transform = new SimpleTransform();
+/* Topics that fire the transform function */
+transform.subscribe(["produce"]);
+/* The strategy the transform engine will use when handling errors */
+transform.errorHandler(PolicyError.SkipOnFailure);
+/* function for validating record has the correct header key and value  */
+const containHeader = (key, value, record) =>
+  record.headers.some( header =>
+    header.headerKey.equals(Buffer.from(key)) &&
+    header.value.equals(Buffer.from(value))
+  )
+
+/* Transform function */
+transform.processRecord((recordBatch) => {
+  console.log("Applying wasm function")
+  return Promise.all(
+    recordBatch.records.map((record) => {
+      if (containHeader("logService", "elastic", record)) {
+        console.log("Index record value to Elastic Search")
+        return client.index({
+          index: "result_wasm",
+          type: "redpanda_wasm",
+          body: {
+            /*
+             index the record value into elastic search
+            */
+            record_message: record.value.toString()
+          }
+        })
+      }
+    })
+  ).then(() => {
+      /*
+       return a map with recordBatch indexed to elastic Search
+      */
+      const result = new Map();
+      result.set("elasticIndex", recordBatch)
+      /* 
+       Wasm processRecord function has to return a Map always,
+       although that map is empty map
+      */
+      return result
+  })
+    .catch((e) => {
+      /* 
+        handling error by coprocessor promise and not
+        coprocessor policy error 
+      */
+      const result = new Map();
+      console.log("fail to publish record batch to Elastic Search, " +
+       `record batch info`, recordBatch.header);
+
+      /*
+        saving failed recordbatch index publish 
+      */
+      result.set("elasticIndexFail", recordBatch)
+      return result
+    })
+});
+
+exports["default"] = transform;
+```
+
+after applying the new logic, we need to compact everything into our 
+wasm function, for it, we can use `npm run build`, this command
+generate `dist/elastic.js` file.
+
+before deploying this wasm function, we need to create the `produce` topic
+which is where we get the records.
+
+```bash
+rpk topic create produce
+```
+
+## Deploy wasm function to Redpanda
+
+we use `rpk wasm deploy <wasm_definition_file_dist_path>` in this case
```

Comment:
```suggestion
`$ rpk wasm deploy <wasm_definition_file_dist_path>`

Reply:
Related to this: I think it's easier when you just tell people exactly what to do. That is, you create a dir with a specific name, in a specific location, then create a file with a specific name, etc. It will prevent new users from making mistakes or typos, and advanced users will know that they can adjust it to their liking.